packages/firebase_ai/firebase_ai/lib/src/content.dart

```diff
@@ -81,7 +83,14 @@ Content parseContent(Object jsonObject) {
 
 /// Parse the [Part] from json object.
 Part parsePart(Object? jsonObject) {
-  if (jsonObject is Map && jsonObject.containsKey('functionCall')) {
+  if (jsonObject is! Map<String, Object?>) {
+    log('Unhandled part format: $jsonObject');
```

Comment:
are the logs necessary if returning the unknown part?

Reply:
Yes, since this is backend added some new feature / Part data that the client SDK might not able to catch up, which is an abnormal case. We would like this information is delivered to developers and the client SDKs would like to catch up the new features going forward.